attributes/default.rb

```diff
@@ -5,6 +5,7 @@
 
 default['hive2']['user']                    = node['install']['user'].empty? ? "hive" : node['install']['user']
 default['hive2']['group']                   = node['install']['user'].empty? ? node['hops']['group'] : node['install']['user']
+default['hive2']['user-home']               = "/home/#{node['hive2']['user']}"
```

Comment:
do we ever want to override this?

Reply:
Most probably not, that's why I don't export also in other cookbooks. I can do it in the weird scenario we would like to have a different home